pages/index.js

```diff
@@ -1,58 +1,128 @@
-import Nav from '../components/Nav'
-import TodoListItem from '../components/TodoListItem'
-import AddTask from '../components/AddTask'
-import { useState, useEffect } from 'react'
-import axios from 'axios'
+import Nav from "../components/Nav";
+import TodoListItem from "../components/TodoListItem";
+import AddTask from "../components/AddTask";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { useAuthRequired } from "../middlewares/auth_required";
+import Script from "next/script";
+import { useAppContext } from "../context/AppContext";
 
 export default function Home() {
-  const API_BASE_URL = 'https://todo-app-csoc.herokuapp.com/'
+    useAuthRequired();
+    const app = useAppContext();
+    const API_BASE_URL = "https://todo-app-csoc.herokuapp.com/";
+    const [profileName, setProfileName] = useState("");
+    const [avatarImage, setAvatarImage] = useState("");
+    const [taskList, setTaskList] = useState([]);
 
-  const [profileName, setProfileName] = useState('')
-  const [avatarImage, setAvatarImage] = useState('')
+    function getTasks() {
+        try {
+            iziToast.destroy();
+            iziToast.info({
+                title: "Welcome",
+                message: "Loading all todos"
+            });
+        } catch (e) {
+            console.error(e);
```

Comment:
You can make use of Error Toast instead of console logging

Reply:
I did replace console logs with iziToast.error in other files, must have missed in this file.